simpleclient_common/pom.xml

```diff
@@ -45,5 +45,11 @@
             <version>4.13.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.3.0</version>
+            <scope>compile</scope>
```

Comment:
I suspect this is only needed for compilation with Java 11 and later. Perhaps it should be marked as optional or could even be removed entirely?

Reply:
Java 11 is required for compilation, so scope = `compile` is required.

Reply:
In that case, I think it should be marked as optional or provided as it shouldn't be needed at runtime.